src/commands/prepare.ts

```diff
@@ -286,10 +274,9 @@ async function checkGitState(
     );
   }
 
-  if (repoStatus.ahead > 0) {
```

Comment:
Removed this check as the way we did it was not reliable anyway: to be able to detect this, we need to run `git fetch` before which we don't.